src/tribol/geom/GeomUtilities.cpp

```diff
@@ -692,14 +705,15 @@ FaceGeomError Intersection2DPolygon( const real* const RESTRICT xA,
 
    // determine the maximum number of intersection points
 
-   // maximum number of vertices between the two polygons. 
+   // maximum number of vertices between the two polygons.
    int maxSegInter = numVertexMax * numVertexMax;
```

Comment:
Is this used anymore?

Reply:
I updated these limits in this routine; verify they make sense.